Content.Server/Chat/ChatSystem.cs

```diff
@@ -99,6 +99,10 @@ private void OnLoocEnabledChanged(bool val)
             SendEntityEmote(source, emoteStr, hideChat);
         }
 
+        // This can happen if the entire string is sanitized out.
+        if (string.IsNullOrEmpty(message))
+            return;
```

Comment:
maybe IsNullOrWhitespace? Or does it already trim whitespace?

Reply:
already trims